tools/nngcat/nngcat_stdin_pipe_test.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
```

Comment:
can we use `#!/usr/bin/env bash` here to facilitate testing on a wider group of unix variants?

Reply:
I was just following the example in the other nngcat test files (e.g. `nngcat_pubsub_test.h`) - probable worth running it by @gdamore

Reply:
Yes, I think its probably a good idea to do this.  We should probably fix the others too.

Really, I'd like to move away from using a shell script to drive these tests, and I have a plan to do that at some point.